libs/types-common/src/Data/Json/Util.hs

```diff
@@ -67,14 +68,14 @@ import qualified Data.Swagger as S
 import qualified Data.Text as Text
 import qualified Data.Text.Encoding as Text
 import qualified Data.Text.Encoding.Error as Text
+-- for UTCTime
```

Comment:
explicit import, then?

Reply:
oh, i just realized this has moved a little.  is it in the right place now or was it before?  either way it's maybe dangerous to have comments in import lists rendered by ormolu?

Reply:
I'll remove the comment. When we start to add such things, where does it end... Also, HLS can show this information.